### PR TITLE
py3(django): fix Django 1.9 deprecation warnings

### PR DESCRIPTION
**Pull request by joshuarli**
Related to https://github.com/getsentry/sentry/pull/15025.

Also updated a bit of the make/test stuff so I could actually run tests, haha.

EDIT: Oh, well there are actually no tests. Anyways, made those changes, might as well keep them.

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -5,10 +5,6 @@
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__)))
 
-# Run tests against sqlite for simplicity
-import os
-os.environ.setdefault('DB', 'sqlite')
```
Comment (wedamija) [inline conftest.py:10]: lol

**Review by wedamija — APPROVED** [review]

**Comment by PaarthB** [comment]
@joshuarli I am running sentry-onpremise 9.1.2. When I try to get this package now (I get it from master), it fails with the error: 
```
File "/usr/local/lib/python2.7/site-packages/sentry_auth_saml2/views.py", line 2, in <module>
    from sentry_auth_saml2.forms import process_metadata
  File "/usr/local/lib/python2.7/site-packages/sentry_auth_saml2/forms.py", line 4, in <module>
    from django.forms.utils import ErrorList
ImportError: No module named utils
```
I believe the on-premise installation is still referring an old version of django?

**Comment by wedamija** [comment]
@PaarthB For the moment you can't run latest master if you're running sentry-onpremise, you should stick with the 9.1.x line.

**Comment by joshuarli** [comment]
@PaarthB Compat with sentry 9.1.x (which is on Django 1.6) has been fixed in https://github.com/getsentry/sentry-auth-saml2/pull/46. So sorry for the delay.

**Comment by PaarthB** [comment]
@joshuarli Will sentry 9.1.x ever be upgraded to newer django version?

**Comment by joshuarli** [comment]
I don't believe we're releasing any more 9.x, although _at least_ Django 1.8 (what sentry master is currently on) will be part of the 10.x release.

**Comment by PaarthB** [comment]
Okay, thanks for the helpful info!

If you have time could you please also respond to the comment I made in this PR: https://github.com/getsentry/sentry/pull/6510

If you are not aware of this, can someone else respond to this?

**Pull request merged**